src/components/Menu/Menu.tsx

```diff
@@ -56,24 +68,33 @@ export const Menu = ({ className, ...other }: MenuProps) => {
 
 /**
  * A button that when clicked, shows or hides the Options
+ * Refer to Button documentation for more style variants.
  */
-const MenuButton = ({ children, className, ...other }: MenuButtonProps) => {
+const MenuButton = ({
+  children,
+  className,
+  showExpandIcon = true,
+  status,
+  ...other
+}: MenuButtonProps) => {
   const buttonClassNames = clsx(styles['menu__button'], className);
   return (
     <HeadlessMenu.Button
-      as={ClickableStyle}
+      as={Button}
       className={buttonClassNames}
-      status="neutral"
+      status={status || 'neutral'}
```

Comment:
do you need to pass `status` for the GST use case? I'm not convinced we want to open the color up more, but lmk what other cases you had in mind

Reply:
Agree that we don't need to open up the color more... same amount of specificity would be required for custom styling whether it's passed or not, so will just set to "neutral"